src/pandablocks_ioc/_tables.py

```diff
@@ -132,15 +141,22 @@ def pack(
         Args:
             row_words: The number of 32-bit words per row
             table_fields_records: The list of fields and their associated RecordInfo
-                structure, used to access the value of each record. The fields and
-                records must be in bit-ascending order (i.e. lowest bit_low field first)
+                structure, used to access the value of each record.
 
         Returns:
             List[str]: The list of data ready to be sent to PandA
         """
 
         packed = None
 
+        # Ensure fields are in bit-order
+        table_fields_records = dict(
+            sorted(
+                table_fields_records.items(),
+                key=lambda item: item[1].field.bit_low,
+            )
+        )
```

Comment:
This code is duplicating lines 91 through 97. Probably worth moving it to its own function.